src/test/store/actions.test.js

```diff
@@ -55,26 +51,10 @@ describe('selectors/getStackTimingByDepthForStackChart', function() {
     ]);
   });
 
-  it('computes "Hide platform details" stack timing by depth', function() {
-    const store = storeWithProfile();
-    store.dispatch(changeHidePlatformDetails(true));
```

Comment:
now that the "implementationFilter" impacts the stack graph too (instead of "hidePlatformDetails"), could you please add a test to check this?